packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -219,10 +226,22 @@ export class Choice extends React.PureComponent<
       'ds-c-choice--small': size === 'small',
     });
 
+    let errorElement;
+    if (errorMessage) {
+      errorElement = (
+        <FieldError id={this.errorId} inversed={inversed} className={errorMessageClassName}>
+          {errorMessage}
+        </FieldError>
+      );
+    }
+
     // Remove props we have our own implementations for
     if (inputProps.id) delete inputProps.id;
     if (inputProps.onChange) delete inputProps.onChange;
 
+    inputProps['aria-describedby'] =
+      mergeIds(this.errorId, this.hintId, inputProps['aria-describedby']) || undefined;
```

Comment:
All the fields that have multiple inputs (ChoiceList, MonthPicker, MultiInputDateField) need to pass down the root `aria-describedby` attributes for the root hint and error messages down to the individual inputs. Dan has tested this

Reply:
This comment might be worth putting into the code as an FYI